rabbit_monitor.py

```diff
@@ -46,13 +50,18 @@
         'messages': message_url,
         }
 
+params = {'lengths_age': settings.stats_window,
```

Comment:
Is "params" a good name? I know it's used not far below but doesn't have much context